src/main/scala/chisel3/util/TransitName.scala

```diff
@@ -42,6 +42,12 @@ object TransitName {
     * @param to the thing that will receive the "good" name
     * @return the `from` parameter
     */
+  @deprecated(
+    "Use suggestName or rely on the naming plugin instead of this function: \n" +
+      "  val from = {to}\n" +
+      "  val from = prefix(prefixYouWant){to}",
+    "Chisel 3.5.5"
```

Comment:
```suggestion
    "Chisel 3.5.4"
```
We haven't released 3.5.4 yet.